mod/thewire/views/default/thewire.js

```diff
@@ -27,15 +27,16 @@ elgg.thewire.textCounter = function(textarea, status, limit) {
 
 	var remaining_chars = limit - $(textarea).val().length;
 	status.html(remaining_chars);
+	var $submit = $(textarea).closest('.elgg-form-thewire-add').find('#thewire-submit-button').eq(0);
```

Comment:
If I am not mistaken selecting by id always returns one element. You would
have to use `[id=x]` to get them all.
On Mar 17, 2016 5:44 PM, "Jeroen Dalsem" <notifications@github.com> wrote:

> In mod/thewire/views/default/thewire.js
> <https://github.com/Elgg/Elgg/pull/9500#discussion_r56536561>:
>
> > @@ -27,15 +27,16 @@ elgg.thewire.textCounter = function(textarea, status, limit) {
> >
> >  	var remaining_chars = limit - $(textarea).val().length;
> >  	status.html(remaining_chars);
> > +	var $submit = $(textarea).closest('.elgg-form-thewire-add').find('#thewire-submit-button').eq(0);
>
> i know the use cases you are referring but you could also have multiple
> id's, however less likely than multiple spans. Luckily both are in control
> of the thewire plugin in this case, so we should be save... i will remove
> them from both locations
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/Elgg/Elgg/pull/9500/files/dac499e865def49425c9d132eef1dcfcd34af61d#r56536561>
>